release_notes.md

```diff
@@ -1,7 +1,4 @@
 # What's New
 
-- This solution pack is now **Certified**
-- **Summary** tab contains more details and now has following sections available as part of this solution pack's details:
-    - A new category **Authentication** &ndash; helps with filtering from the left pane when using the check-boxes under **Category**
-    - A new tag **Brute Force Attack** &ndash; helps with filtering from the left pane when using the **Search By Tags** option
-- The description of the scenario **Brute Force Attempt (FortiSIEM)** is now more comprehensive
+- Modified `Investigate Brute Force Attempt (FortiSIEM)` and `Investigate Brute Force Attempt` playbooks
+    - Updated jinja logic to check malicious IP Address
```

Comment:
This description is not enough. Please add more details

Reply:
Updated the release notes, please check